docs/IntegrationWithExistingApps.md

```diff
@@ -782,3 +797,13 @@ if (!foundHash) {
   display('platform', isMac ? 'objc' : 'android');
 }
 </script>
+
+## Creating production build in Android Studio
+
+You can use Android Studio to create your release builds too! It’s as easy as creating release builds of your previously-existing native Android app. There’s just one additional step, which you’ll have to do before every release build. You need to execute the following to create a React Native bundle, which’ll be included with your native Android app:
```

Comment:
I've tried merging a gradle script file for an existing project with the `react.gradle` one from `react-native`, but without success. There is too much going on and I'm not a gradle expert.